src/script/interpreter.h

```diff
@@ -76,6 +76,10 @@ enum
     // (softfork safe, BIP62 rule 6)
     // Note: CLEANSTACK should never be used without P2SH.
     SCRIPT_VERIFY_CLEANSTACK = (1U << 8),
+
+    // Verify CHECKLOCKTIMEVERIFY (BIP65)
+    //
```

Comment:
Stupidest little nit... this empty comment line can be removed, no? Or better yet explain the feature like the other comments do.